supabase/updateData.ts

```diff
@@ -160,6 +172,8 @@ export const saveData = async ({
 			message_id: message.id,
 			url: new URL(attachment.url).pathname,
 			content_type: attachment.content_type,
+			width: attachment.width,
```

Comment:
Does Discord provide us with the width/height of an attachment (if it's an image)?